/**:
  ros__parameters:
    update_rate: 10.0
    add_duplicated_node_names_to_msg: true
    nodes_to_ignore:  # List of nodes to ignore
      - /rviz
      - /talker
